src/Feature/Keyboard/Feature_Keyboard.re

```diff
@@ -0,0 +1,77 @@
+open Oni_Core;
+
+// Placeholder model type
+type model = int;
+let initial: model = 0;
+
+[@deriving show]
+type command =
+  | ShowEmojiAndSymbols;
+
+[@deriving show]
+type msg =
+  | Empty(unit)
```

Comment:
We could change this to `Empty` with no args - that would simplify it below (We use `Noop` though in some other parts of the code).